README.md

```diff
@@ -17,17 +17,16 @@ or controller layer, you're probably not looking for Ransack.
 
 Ransack is supported for Rails 7.0, 6.x on Ruby 2.6.6 and later.
 
-In your Gemfile, for the last officially released gem:
+Run to install:
```

Comment:
Can we say that the command below also adds the gem to the `Gemfile` automatically? Maybe something like

```suggestion
To install `ransack` and add it to your Gemfile, run
```